giraffe/src/types/index.ts

```diff
@@ -58,8 +58,8 @@ export interface Config {
   legendCrosshairColor?: string
   legendColumns?: string[]
 
-  // The type of the y-axis column
-  yColumnType?: ColumnType
+  // The labels for the y-axis column
```

Comment:
✂️  - i think this comment has become unnecessary because `yColumnLabels` is pretty darn self explanatory